com.unity.multiplayer.mlapi/Runtime/Transports/UNET/UNetTransport.cs

```diff
@@ -64,6 +64,17 @@ public enum SendMode
         private SocketTask m_ConnectTask;
         public override ulong ServerClientId => GetMLAPIClientId(0, 0, true);
 
+        #if UNITY_EDITOR
```

Comment:
super-nitpick: shouldn't these macros be on the left with no spaces in front of them?
do you mind formatting the code in an IDE? 😛

Reply:
Should they?

I do use an IDE for my development.

Reply:
huh, looking at all other places and testing locally in vscode, all macros are being formatted and sticking to the left

Reply:
anyways...